src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -103,6 +103,8 @@ object DeployPlugin extends AutoPlugin {
     // Clients may override to perform a different action in addition to staging (i.e. running
     // database migrations).
     val preDeploy = taskKey[Unit]("Prep the project for a deploy. Defaults to `stageAndCacheKey`.")
+
+    val deployNpmBuild = taskKey[Unit]("Runs the npm build")
```

Comment:
Yeah, that is my least favorite part of this PR. I'd prefer to have a config like `npmBuild in Deploy`, but we didn't have that set up already